mldsa/symmetric-shake.c

```diff
@@ -12,8 +12,9 @@ void mldsa_shake128_stream_init(keccak_state *state,
                                 uint16_t nonce)
 {
   uint8_t t[2];
-  t[0] = nonce;
-  t[1] = nonce >> 8;
+
+  t[0] = nonce & 0xFF;       
+  t[1] = (nonce >> 8) & 0xFF;
```

Comment:
The second truncation is not needed since the value is already within the range of `uint8_t`.